packages/suite/src/views/view-only/ViewOnlyTooltip.tsx

```diff
@@ -1,27 +1,50 @@
-import { Button, Text, Tooltip } from '@trezor/components';
-import { spacingsPx, zIndices } from '@trezor/theme';
+import { Button, Icon, Text, Tooltip } from '@trezor/components';
+import { borders, palette, spacingsPx, zIndices } from '@trezor/theme';
 import { useDispatch, useSelector } from 'src/hooks/suite';
 import { selectSuiteFlags } from 'src/reducers/suite/suiteReducer';
-import styled from 'styled-components';
+import styled, { useTheme } from 'styled-components';
 import { setFlag } from 'src/actions/suite/suiteActions';
 import { Translation } from 'src/components/suite';
 
 type ViewOnlyTooltipProps = {
     children: React.ReactNode;
 };
 
+const Icons = styled.div`
+    display: flex;
+    flex-direction: row;
+`;
+
+const IconContainer = styled.div`
+    border: solid 1px ${palette.lightWhiteAlpha400};
+    background-color: ${palette.darkGray300};
+    border-radius: ${borders.radii.full};
+    padding: 4px;
```

Comment:
is this 4 connected to this: 

![image](https://github.com/trezor/trezor-suite/assets/152899911/8bf7970f-f47f-4719-a379-4b0d2af58c24)


If yes lets create some constant that explains it.

Or use `spacings` constants? Not sure if it makes semantically sense here.